utils/remote_control/navigator_emergency_control/nodes/nodes/navigator_emergency.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+
+from remote_control_lib import RemoteControl
+import rospy
+from sensor_msgs.msg import Joy
+
+
+__maintainer__ = "David Zobel"
+__email__ = "zobeldavid@ufl.edu"
+__copyright__ = "Copyright 2016, MIL"
+__license__ = "MIT"
+
+
+rospy.init_node("emergency")
+
+
+class Joystick(object):
+
+    def __init__(self):
+        self.force_scale = rospy.get_param("~force_scale", 600)
+        self.torque_scale = rospy.get_param("~torque_scale", 500)
+
+        self.remote = RemoteControl("emergency", "/wrench/emergency")
+        rospy.Subscriber("joy_emergency", Joy, self.joy_recieved)
+
+        self.active = False
+        self.reset()
+
+    def reset(self):
+        '''
+        Used to reset the state of the controller. Sometimes when it
+        disconnects then comes back online, the settings are all out of whack.
+        '''
+        self.last_kill = False
+        self.last_station_hold_state = False
+        self.last_auto_control = False
+        self.last_emergency_control = False
+        self.last_shooter_cancel = False
+        self.last_change_mode = False
+
+        self.start_count = 0
+        self.last_joy = None
+        self.active = False
+
+        self.remote.clear_wrench()
+
+    def check_for_timeout(self, joy):
```

Comment:
This function is replaced by your timeout function, so I would recommend removing it and all reverences to it.